test/verify/files/mock-kube-vm.json

```diff
@@ -0,0 +1,70 @@
+{
```

Comment:
With that you should remove the verbatim copy in the Python test file, and read it from that file instead. Let's avoid maintaining two copies of this.

Reply:
done